### ISSUE TITLE
Update deprecated API

### ISSUE THREAD

**Issue by astudilloalex** [Baseflow/flutter-geocoding#70]
## 🚀 Feature Requests

Compile for Android show the next Error
```
Note: C:\Users\LENOVO\AppData\Local\Pub\Cache\hosted\pub.dartlang.org\geocoding-
2.0.1\android\src\main\java\com\baseflow\geocoding\GeocodingPlugin.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
```
We need to update the API.

### Platforms affected (mark all that apply)
- [ ] :iphone: iOS
- [x] :robot: Android

**Comment by BRSPRO** [comment]
the same issue please help with the solution

**Comment by mvanbeusekom** [comment]
Has been fixed in version 2.0.3.
